### DISCUSSION TITLE
2ο Παραδοτέο - Π2019152

### DISCUSSION THREAD

**Discussion by Tarakhs** [courses-ionio/sw#1277] [Show and tell]
Για αυτό το παραδοτέο ζητήθηκε μια άσκηση γραμμής εντολών πάνω σε ένα σύστημα που χρησιμοποιεί systemd. Επέλεξα να χρησιμοποιήσω το εργαλείο ntfy, και μετά από λίγο testing έγραψα ένα απλό script το οποίο δέχεται ως παράμετρο μια εντολή προς εκτέλεση, και στέλνει το αντίστοιχο notification όταν αυτή ολοκληρωθεί (ή όχι) στο κινητό.

[Link Αναφοράς](https://github.com/Tarakhs/sw/tree/2019152/projects/2019152/README.md) | [Link Asciinema](https://asciinema.org/a/VVuhp2XGYtQUIGiGnSSXiVVkD)

**Comment by p20sami** [comment]
Καλησπέρα, θα πρότεινα να βάλεις το ΑΜ σου στο command prompt ώστε να φαίνεται. Πολύ καλή δουλειά πέραν αυτού!
